.test.mk

```diff
@@ -90,7 +90,8 @@ CMAKE_PARAMS_ASAN = -DENABLE_WERROR=ON \
                     -DENABLE_UB_SANITIZER=ON \
                     -DFUZZING_BUILD_MODE_UNSAFE_FOR_PRODUCTION=OFF \
                     -DENABLE_FUZZER=ON \
-                    -DTEST_BUILD=ON
+                    -DTEST_BUILD=ON \
+		    -DLUAJIT_ENABLE_GC64=ON
```

Comment:
I suggest adding `-DLUAJIT_USE_SYSMALLOC=ON` to avoid [this warning](https://github.com/tarantool/tarantool/actions/runs/9969351885/job/27546159031?pr=10232#step:8:188), unless the ASAN support for the LuaJIT internal allocator is complete.

Reply:
This PR has been changed and the comment is now irrelevant. I'll let it be resolved.